src/app/archive/create_schema.sql

```diff
@@ -19,7 +19,7 @@ CREATE TABLE blocks (
   ledger_proof_nonce int NOT NULL,
   status int NOT NULL,
   block_length int NOT NULL,
-  block_time timestamp NOT NULL
+  block_time bit(64) NOT NULL
 );
```

Comment:
according to the RFC comments, do we want this field to be called time_received?